### PR TITLE
feat: Correct Logic

### PR DESCRIPTION
**Pull request by AashishChakravarty**

### CODE REVIEW

**Comment by zoedsoupe** [comment]
Wow thank you so much! I was freaking out with this challenge... ≈

File: lib/war.ex [hunk 1]
```diff
@@ -12,6 +12,7 @@ defmodule War do
     {p1, p2} =
       deck
       |> Enum.map(&apply_ace_weight/1)
+      |> Enum.reverse()
```
Comment (zoedsoupe) [inline lib/war.ex:15]: But I didn't understand why needs to reverse the deck before splitting it. Could you please elaborate on it?

Comment (AashishChakravarty) [reply lib/war.ex:15]: You can also reverse both files after split. In real world we always play from top card not the last so  we need to reverse

Comment (zoedsoupe) [reply lib/war.ex:15]: Wow, that makes a lot of sense!

**Review by zoedsoupe — APPROVED** [review]
Thank you so much for your contribution!

**Pull request merged**